runtime/astar/src/lib.rs

```diff
@@ -782,6 +784,13 @@ parameter_types! {
     );
     pub PrecompilesValue: Precompiles = AstarNetworkPrecompiles::<_, _>::new();
     pub WeightPerGas: Weight = Weight::from_parts(WEIGHT_PER_GAS, 0);
+    /// The amount of gas per PoV size. Value is calculated as:
+    ///
+    /// max_gas_limit = max_tx_ref_time / WEIGHT_PER_GAS = max_pov_size * gas_limit_pov_size_ratio
+    /// gas_limit_pov_size_ratio = ceil((max_tx_ref_time / WEIGHT_PER_GAS) / max_pov_size)
+    ///
+    /// Equals 4 for values used by Astar runtime.
+    pub const GasLimitPovSizeRatio: u64 = 4;
```

Comment:
Is it possible to evaluate this value during compile time? Since the calculation is depended on several variables that might change and we might miss to update this.

Reply:
I could do it via Fixed point u128, but I'd rather keep as it is.

I expect this to be changed very soon, since we'll also be charging for PoV size as part of regular tx payment.
And there are ongoing activities to also incorporate ref time measurement together with PoV to get a better gas estimate.

Change to max ref time or PoV size isn't coming any time soon though.